src/uct/api/uct.h

```diff
@@ -1311,11 +1329,28 @@ UCT_INLINE_API void uct_ep_pending_purge(uct_ep_h ep, uct_pending_callback_t cb)
 
 /**
  * @ingroup UCT_RESOURCE
- * @brief
- */
-UCT_INLINE_API ucs_status_t uct_ep_flush(uct_ep_h ep)
+ * @brief Flush outstanding communication operations on an endpoint.
+ *
```

Comment:
Hmm. I don't remember what we have decided. I think on UCT level it also flashes AM ?